gunagala/imager.py

```diff
@@ -1346,19 +1346,19 @@ def exp_time_sequence(self,
         if filter_name not in self.filter_names:
             raise ValueError("This Imager has no filter '{}'!".format(filter_name))
 
-        if bool(bright_limit) == bool(shortest_exp_time):
+        if (bright_limit is not None and shortest_exp_time is not None) or (bright_limit is None and shortest_exp_time is None):
```

Comment:
I feel the ValueError message is not for the second condition after "or".

Perhaps there should be another "if" block to raise ValueError when neither of bright_limit and shortest_exp_time are provided and have a different message that let the user know one of the two should be provided?